runtime/environment-cpu.yml

```diff
@@ -19,3 +19,6 @@ dependencies:
   - pip:
       - flwr[simulation]==1.1.0
       - tensorflow-cpu==2.10.0
+      - igraph==0.10.3
+      - argparse==1.4.0
+      - python-time==0.3.0
```

Comment:
`python-time` looks like an unmaintained and uncommon package, which has a strict pin on `arrow==0.14.2` which is a very old version. It's also a very small amount of code. 

In order for easier dependency resolution in case of other dependencies on arrow, it would be best if you use arrow directly and instead add a dependency on a more recent version of arrow. It should be pretty easy to vendor the source code from this package: https://github.com/hengdai/pytime